python/src/ir.cc

```diff
@@ -1575,6 +1575,13 @@ void init_triton_ir(py::module &&m) {
         if (triton::tools::getBoolEnv("TRITON_ENABLE_LLVM_DEBUG")) {
           ::llvm::DebugFlag = true;
         }
+
+        auto debugOnly = triton::tools::getenv("TRITON_LLVM_DEBUG_ONLY");
+        if (!debugOnly.empty()) {
+          ::llvm::DebugFlag = true;
+          ::llvm::setCurrentDebugType(debugOnly.c_str());
```

Comment:
Will it be better to use setCurrentDebugTypes so we can use a list of passes?

Reply:
I can make the env var a comma separated string of pass names, will update soon.

Reply:
Updated, WDYT?